notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,34 @@
 {
```

Comment:
<h2>Intro edits:</h2><p>"When analyzing structures it is crucial to understand the mechanics keeping it stable. Applied forces on a floor, beam, or structure create reaction forces and moments. These reactions are the structure resisting movement without breaking. You will look at structures that do not move despite having forces applied to them. In this case Newton's second law states that both the acceleration and sum of forces in the system must be zero. Newton's second law describes the sum of forces in all directions. The reactions present must be equal and opposite to sum of forces. You can represent and solve this concept with NumPy arrays. "</p><p> </p><p>rewrite from:</p><h2>[Hemingwayapp](https://hemingwayapp.com/)</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>